tests/andromede/test_ac_link.py

```diff
@@ -0,0 +1,385 @@
+        ModelPort(port_type=AC_PORT, port_name="port2"),
+    ],
+    port_fields_definitions=[
+        PortFieldDefinition(
+            port_field=PortFieldId("port1", "flow"),
+            definition=-var("flow"),
+        ),
+        PortFieldDefinition(
+            port_field=PortFieldId("port2", "flow"),
+            definition=var("flow"),
+        ),
+    ],
+    binding_constraints=[
+        Constraint(
+            name="AC flow",
+            expression=var("flow")
+            == 1
+            / param("reactance")
+            * (port_field("port1", "angle") - port_field("port2", "angle")),
+        )
+    ],
+)
+
+"""
+Flow on the line is proportional to angle difference between extremities,
+and inverse of impedance.
+"""
+AC_LINK_WITH_LIMIT = model(
+    id="LINK",
+    parameters=[
+        float_parameter("reactance", IndexingStructure(False, False)),
+        float_parameter("flow_limit", IndexingStructure(False, False)),
+    ],
+    variables=[float_variable("flow", -param("flow_limit"), param("flow_limit"))],
+    ports=[
+        ModelPort(port_type=AC_PORT, port_name="port1"),
+        ModelPort(port_type=AC_PORT, port_name="port2"),
+    ],
+    port_fields_definitions=[
+        PortFieldDefinition(
+            port_field=PortFieldId("port1", "flow"),
+            definition=-var("flow"),
+        ),
+        PortFieldDefinition(
+            port_field=PortFieldId("port2", "flow"),
+            definition=var("flow"),
+        ),
+    ],
+    binding_constraints=[
+        Constraint(
+            name="AC flow",
+            expression=var("flow")
+            == 1
+            / param("reactance")
+            * (port_field("port1", "angle") - port_field("port2", "angle")),
+        )
+    ],
+)
+
+"""
+Flow on the line is proportional to angle difference between extremities,
+and inverse of impedance.
+A shift angle is applied on side 1, allowing to control the flow.
+"""
+AC_LINK_WITH_PST = model(
+    id="LINK_WITH_PST",
+    parameters=[float_parameter("reactance", IndexingStructure(False, False))],
+    variables=[float_variable("flow"), float_variable("shift")],
+    ports=[
+        ModelPort(port_type=AC_PORT, port_name="port1"),
+        ModelPort(port_type=AC_PORT, port_name="port2"),
+    ],
+    port_fields_definitions=[
+        PortFieldDefinition(
+            port_field=PortFieldId("port1", "flow"),
+            definition=-var("flow"),
+        ),
+        PortFieldDefinition(
+            port_field=PortFieldId("port2", "flow"),
+            definition=var("flow"),
+        ),
+    ],
+    binding_constraints=[
+        Constraint(
+            name="AC flow",
+            expression=var("flow")
+            == 1
+            / param("reactance")
+            * (
+                port_field("port1", "angle")
+                + var("shift")
+                - port_field("port2", "angle")
+            ),
+        )
+    ],
+)
+
+
+def test_ac_network_no_links():
```

Comment:
Small description of the test case and numerical values to convince the reader that the optimum is indeed found ? The remark applies to all test cases

Reply:
I have added some information to each test case, tell me if it looks OK or if you see possible improvements